generator/internal/format_class_comments.h

```diff
@@ -22,12 +22,21 @@ namespace google {
 namespace cloud {
 namespace generator_internal {
 
-/// The function does not just use service.name() and takes @p service_name in
-/// case there exists a service_name_mapping argument that is renaming the
-/// service.
+/** The function formats class comments based on the pre-existing service
```

Comment:
Typically the comment text starts on the next line.

Reply:
Done 

Addressed in https://github.com/googleapis/google-cloud-cpp/pull/12983